iroh/src/magicsock/node_map/node_state.rs

```diff
@@ -334,6 +334,7 @@ impl NodeState {
                 Level::DEBUG,
                 remote_node = %self.node_id.fmt_short(),
                 conn_type = ?typ,
+                have_ipv6,
```

Comment:
is this relevant here?

Reply:
I think so. Because if `have_ipv6` is false, this affects the choice of connection type.

I can see this being a little too noisy. Let me know if I should remove this.

Reply:
You appear to have removed it from the trace logs but kept it in the event.  I questioned more whether it was useful in the event, trace logs are just trace logs.  I'm not sure it changes the meaning of the event whether that's there or not.  But in any case, logs and events are living things, we tweak them when we see fit.  So if you think this helps, then add it.  I was only checking you found this helpful on the event because I wasn't sure what extra info it gave.

Reply:
:facepalm: Sorry I simply edited the wrong lines. Thanks for verifying!